addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -206,7 +205,15 @@ function align(editor, mode) {
             if (!visitedBlocks.has(block)) {
                 const hasModifier = getComputedStyle(block).textAlign === mode;
                 if (!hasModifier && block.isContentEditable) {
-                    block.oAlign(sel.anchorOffset, mode);
+                    block.oAlign(mode);
+                    const link = closestElement(node, 'a');
+                    if (link) {
+                        // Since we apply contentEditable true to links when the
+                        // selection is in them, make sure to apply the
+                        // alignment to it as well so as not to throw
+                        // queryCommandState off.
+                        link.oAlign(mode);
```

Comment:
This whole could maybe be done to `[contenteditable]` instead to be more generic. I don't know what's best.

Reply:
I'd rather keep it on links alone for now to limit the potential damage in case of a regression. If we see other cases where we have the same issue and we realize we can in fact generalize it to `[contenteditable]` then we'll do it at that point.